### DISCUSSION TITLE
Feature requests

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#569] [General]
Hey hey,

Let's centralize all feature requests under the [💫 Feature requests](https://github.com/avo-hq/avo/discussions/categories/feature-requests) category.

### Proposal 📣
You can propose new ones with the level of detail you feel comfortable with and we can refine them afterward. 

### Voting 👍 👎
Then, others can vote on them and we'll pick them up in the next development cycle.

Go and submit [a new feature request](https://github.com/avo-hq/avo/discussions/new?category=feature-requests).

Thank you guys and gals 🤙

### Discussions are not items in progress

To make it easier to follow items in progress, we're only going to consider [Issues](https://github.com/avo-hq/avo/issues) as items in progress. Discussions are only here to talk about features and ideas. When something will get picked up to work on, it will be converted to a ticket.

**Comment by prohysio876** [comment]
Hello, I am seeking someone to make a CRM using avo. any full stack developer available?

**Reply by adrianthedev** [reply]
Hey @prohysio876. Send me an email (adrian@avohq.io) with the project description and maybe we can find someone.

**Comment by prohysio876** [comment]
Hello,
  I am not a developer/fullstack or etc, I run a physiotherapy company, I
need a CRM for my custom EMR (built of Ruby) and was directed to your page.
My requirements for the CRM are :
- Lead capture and conversion
- Communication tracking (calls, notes, follow-ups)
- Task management and automation
- Interaction history (timeline)
- Integration with Accounts Receivable (AR)
- Integration with Patient Profile (EMR)
-Integration with appointment
- Integration with Grandstream CDR (calls)

CRM Module Responsible for:

- Leads
- Contact management
- Interaction tracking
- Calls
- Tasks
- Automations
- Follow-ups

Let me know how we would work
Noel Brown
Physiotherapist
Bsc. Physical Therapy (Hons)
CNMT | CFIDN | MFR | FAKTR
"The results you want. The care you deserve."
This message (including any attachments) is intended only for the use of
the individual or entity to which it is addressed and may contain
information that is non-public, proprietary, privileged, confidential, and
exempt from disclosure under applicable law or may constitute as attorney
work product. If you are not the intended recipient, you are hereby
notified that any use, dissemination, distribution, or copying of this
communication is strictly prohibited. If you have received this
communication in error, notify us immediately by telephone and (i) destroy
this message if a facsimile or (ii) delete this message immediately if this
is an electronic communication. Thank you.
